docs/memo.md

```diff
@@ -0,0 +1,189 @@
+# Memo
+
+## Dependency
+It is available in general package `"ru.tinkoff" %% "tofu" % Versions.tofu` and as standalone dependency `"ru.tinkoff" %% "tofu-memo" % Versions.tofu`
+
+## Functionality
+* A cache of a single value on access.
+* A cache of a values mapping on access.
+* TTL. Expired values are discarded on access.
+* Forced invalidation based on time.
+
+Missed:
+* Infinite caching (workaround is ttl `FiniteDuration(Long.MaxValue, TimeUnit.NANOSECONDS)` = 292 days).
+* Background renewal of cached values.
+* Forced invalidation of single key (mapping cache).
+
+## Examples
+### Single value cache
+```scala
+import tofu.memo._
+import cats.effect.{Clock, Sync}
+import cats.syntax.applicative._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import monix.eval.Task
+import monix.execution.Scheduler.Implicits.global
+import scala.concurrent.duration.FiniteDuration
+import java.util.concurrent.TimeUnit
+
+def effect[F[_] : Sync]: F[Int] = Sync[F].delay(println("called")) >> 335.pure[F]
```

Comment:
Is `>> 335.pure[F]` intentional? Since you have `cats.syntax.functor._` import, you could write that like `Sync[F].delay(println("called")).as(335)`

Reply:
Changed.